.github/integration/sda/config.yaml

```diff
@@ -68,6 +68,10 @@ c4gh:
   passphrase: "c4ghpass"
   syncPubKeyPath: /shared/sync.pub.pem
 
+grpc:
+  host: "reencrypt"
+  port: "50051"
+
```

Comment:
This file down't need this addition yet.

Reply:
That's correct. I will remove it.

Reply:
Removed in a5baaa6d